dll/win32/shell32/lang/es-ES.rc

```diff
@@ -620,10 +626,10 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBL
 CAPTION "Cambiar icono"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
-    LTEXT "Look for icons in this file:", -1, 7, 7, 179, 10
+    LTEXT "Buscar íconos en este archivo:", -1, 7, 7, 179, 10
```

Comment:
See above (will stop mentioning more instances of this even if they appear later).